Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "ncurses-6.1.tar.gz" "ncurses" "host-ncurses" "add-missing-getopt-include" "\
        progs/clear.c \
        progs/infocmp.c \
        progs/tic.c \
        progs/toe.c \
        progs/tput.c \
        progs/tset.c \
    ";


===============================================================================
--- a/progs/clear.c	2017-10-07 23:48:32.000000000 +0200
+++ b/progs/clear.c	2019-01-02 22:35:05.000000000 +0100
@@ -39,6 +39,7 @@
 #define USE_LIBTINFO
 #include <clear_cmd.h>
 #include <tty_settings.h>
+#include <getopt.h>
 
 MODULE_ID("$Id: clear.c,v 1.22 2017/10/07 21:48:32 tom Exp $")
 
===============================================================================
--- a/progs/infocmp.c	2017-04-05 11:27:51.000000000 +0200
+++ b/progs/infocmp.c	2019-01-02 22:36:35.000000000 +0100
@@ -41,6 +41,7 @@
 #include <progs.priv.h>
 
 #include <dump_entry.h>
+#include <getopt.h>
 
 MODULE_ID("$Id: infocmp.c,v 1.143 2017/04/05 09:27:51 tom Exp $")
 
===============================================================================
--- a/progs/tic.c	2018-01-25 20:16:50.000000000 +0100
+++ b/progs/tic.c	2019-01-02 22:35:05.000000000 +0100
@@ -47,6 +47,7 @@
 #include <hashed_db.h>
 #include <parametrized.h>
 #include <transform.h>
+#include <getopt.h>
 
 MODULE_ID("$Id: tic.c,v 1.252 2018/01/25 19:16:50 tom Exp $")
 
===============================================================================
--- a/progs/toe.c	2017-12-23 20:23:40.000000000 +0100
+++ b/progs/toe.c	2019-01-02 22:35:05.000000000 +0100
@@ -43,6 +43,7 @@
 #if USE_HASHED_DB
 #include <hashed_db.h>
 #endif
+#include <getopt.h>
 
 MODULE_ID("$Id: toe.c,v 1.77 2017/12/23 19:23:40 tom Exp $")
 
===============================================================================
--- a/progs/tput.c	2017-10-14 22:46:43.000000000 +0200
+++ b/progs/tput.c	2019-01-02 22:35:05.000000000 +0100
@@ -49,6 +49,7 @@
 #endif
 #include <transform.h>
 #include <tty_settings.h>
+#include <getopt.h>
 
 MODULE_ID("$Id: tput.c,v 1.78 2017/10/14 20:46:43 tom Exp $")
 
===============================================================================
--- a/progs/tset.c	2017-10-08 02:01:29.000000000 +0200
+++ b/progs/tset.c	2019-01-02 22:35:05.000000000 +0100
@@ -96,6 +96,7 @@
 #ifdef NeXT
 char *ttyname(int fd);
 #endif
+#include <getopt.h>
 
 MODULE_ID("$Id: tset.c,v 1.120 2017/10/08 00:01:29 tom Exp $")
 
